commit 24c03bd379eb02abb5cfd88ce51a3a6714112d2f
Author: James Cloos <cloos@jhcloos.com>
Date:   Wed Aug 20 10:32:49 2008 -0400

    xaw8 is gone, use xaw7

diff --git a/configure.ac b/configure.ac
index a13ead8..3593007 100644
--- a/configure.ac
+++ b/configure.ac
@@ -31,7 +31,7 @@ AM_CONFIG_HEADER(config.h)
 AC_PROG_CC
 AC_PROG_INSTALL
 
-XAW_CHECK_XPRINT_SUPPORT(VIEWRES)
+PKG_CHECK_MODULES(VIEWRES, xaw7)
 
 AC_SUBST(VIEWRES_CFLAGS)
 AC_SUBST(VIEWRES_LIBS)
